### PR TITLE
Reach state queries

### PR DESCRIPTION
**Pull request by sreynen**
No changes to the Python code for this, just a bunch of SQL. And all of these SQL files are the same, just for different states.

### CODE REVIEW

File: reach_voters_az.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
+WITH ak_names AS
+  (SELECT DISTINCT cu.id AS user_id,
+                   LISTAGG(DISTINCT uf.value, ',') WITHIN
```
Comment (bodhello) [inline reach_voters_az.sql:3]: Should the delimiter be `','` or `', '`?

File: reach_voters_az.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+   JOIN ak_moveon.core_action ca ON ca.user_id = cu.id
+   JOIN stafftemp.latest_phone lp ON lp.akid = cu.id
+   JOIN reach.tmc__mvo_users ru ON ru.phonenumber = lp.mobile
+   LEFT JOIN ak_moveon.core_userfield uf ON uf.parent_id = cu.id
+   AND uf.name = 'reach_voters_az'
+   WHERE ca.page_id = 33039
+   GROUP BY 1),
+     reach_names AS
+  (SELECT DISTINCT cu.id AS user_id,
+                   LISTAGG(DISTINCT rr.personfirstname || ' ' || rr.personlastname, ',') WITHIN
```
Comment (bodhello) [inline reach_voters_az.sql:16]: also question about the delimiter here

File: reach_voters_az.sql (hunk 1 above)
Comment (sreynen) [reply reach_voters_az.sql:3]: The delimiter is used by code to split it into a list, specifically https://github.com/MoveOnOrg/ak-userfield-sync/blob/main/sync.py#L52, so while a space would make it more human-readable, humans aren't the ones reading it.

Comment (bodhello) [reply reach_voters_az.sql:3]: 👍

**Pull request merged**